bayes/vb.py

```diff
@@ -329,6 +329,7 @@ def __init__(self, n_trials_max=10, iter_max=50, tolerance=0.1):
         self.n_trials_max = n_trials_max
         self.tolerance = tolerance
         self.iter_max = iter_max
+        self.scale_by_prior_mean = True
```

Comment:
Wouldn't it be better to store the scaling factors that could either be automatically obtained from the prior or directly set (in case the mean is zero). Then the interface would cover both cases and allow users to directly prescribe other values than the prior mean.